src/cljs/org/broadinstitute/firecloud_ui/page/profile.cljs

```diff
@@ -32,13 +36,58 @@
          {:style {:marginTop "0.167em" :width "30ex"}
           :value (get-in @state [:values key])
           :onChange #(swap! state assoc-in [:values key] (-> % .-target .-value))})]])
-   :component-did-mount
+   :render-nih-link-section
    (fn [{:keys [state]}]
+     (let [{:keys [linkedNihUsername lastLinkTime isDbgapAuthorized]} (:values @state)]
+       [:div {}
+        [:h3 {} "Linked NIH Account"]
+        [:div {:style {:display "flex"}}
+         [:div {:style {:flex "0 0 20ex"}} "NIH Username:"]
+         [:div {:style {:flex "0 0 auto"}}
+          (cond
+            (:pending-nih-username-token @state)
+            [components/Spinner {:ref "pending-spinner" :text "Creating NIH account link..."}]
+            (nil? linkedNihUsername)
+            [:a {:href (str (:shibboleth-url-root @state)
+                            "/link-nih-account?redirect-url="
+                            (js/encodeURIComponent
+                             (str (-> js/window .-location .-href)
+                                  "/nih-username-token={token}")))}
+             "Log-In to NIH to link your account"]
+            :else
+            linkedNihUsername)]]
+        [:div {:style {:display "flex" :marginTop "1em"}}
+         [:div {:style {:flex "0 0 20ex"}} "dbGaP Authorization:"]
+         [:div {:style {:flex "0 0 auto"}}
+          (if (nil? linkedNihUsername)
+            [:i {} "N/A"]
+            (if isDbgapAuthorized
```

Comment:
when I set my profile in Thurloe to isDbgapAuthorized=false, this still renders "Authorized" in the UI.

Possibly this is a problem somewhere in js->cljs truthy/falsey? Remember that Thurloe stores and retrieves a string value, so the profile is going to return as:
{
    "key": "isDbgapAuthorized",
    "value": "false"
  }